Formula/gdb.rb

```diff
@@ -3,7 +3,7 @@ class Gdb < Formula
   homepage "https://www.gnu.org/software/gdb/"
   url "https://ftp.gnu.org/gnu/gdb/gdb-10.1.tar.xz"
   mirror "https://ftpmirror.gnu.org/gdb/gdb-10.1.tar.xz"
-  sha256 "f82f1eceeec14a3afa2de8d9b0d3c91d5a3820e23e0a01bbb70ef9f0276b62c0"
+  sha256 "250c419f5130b031328556d66a042173a74ac7f183fa50ed1032aed199eb6499"
```

Comment:
That's supposed to be the SHA-256 sum of the file that `url` (and `mirror`) point to. I just did the download again and I'm finding the same SHA as before (the one that starts with f82f). How did you get this value?

Reply:
I got the new SHA-256 sum by brew tap/downloading from vegz78/gdb/gdb...

Reply:
Strange — I just tried to build out of your repository and I am getting the old SHA-256:

<img width="829" alt="image" src="https://user-images.githubusercontent.com/1629585/123441984-bd430080-d5d4-11eb-8566-51f31f8102ce.png">

Are you perhaps working from behind some kind of proxy or cache? Could you please upload somewhere whatever binary you are getting out of downloading https://ftp.gnu.org/gnu/gdb/gdb-10.1.tar.xz ?

Thanks in advance!

Reply:
I got the same error when I downloaded first time my new vegz78/gdb/gdb tap on top of your old tap in my ~/Library/Caches/Homebrew...  After deleting the cache and downloading again, the SHA-256 sum stayed "250c419..." which I then updated in the gdb.br file.

Reply:
You might even get a 3rd new one when merging my pull request... ;-)